clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -792,6 +794,43 @@ untaken conditional branch:
       801377      601843:     1159769 ifetch       4 byte(s) @ 0x00007fc2c3aa5c72 48 83 c4 48          add    $0x48, %rsp
 \endcode
 
+Here is an example of an asynchronous signal interrupting an instruction:
```

Comment:
Make it clear this is not what someone will see in a real trace!  (At least not a new-version trace.). Otherwise this may be very confusing.